app/lib/question_navigation.rb

```diff
@@ -26,6 +26,8 @@ class QuestionNavigation
 
     # Consent
     Questions::ConsentController, # Advances statuses to "In Progress"
```

Comment:
These comments make what's going on in each question action / after update easier to understand. Thanks!